crates/uv/src/commands/project/run.rs

```diff
@@ -185,7 +185,7 @@ pub(crate) async fn run(
 }
 
 /// Update a [`PythonEnvironment`] to satisfy a set of [`RequirementsSource`]s.
-async fn update_environment(
+pub(crate) async fn update_environment(
```

Comment:
@charliermarsh where do you think we should put this?

Reply:
Probably in the module root.

Reply:
Maybe we should have environment update code in the `virtualenv` crate or some sort of dedicated crate for managing environments?

Reply:
Moving this to #3659